src/main/java/seedu/commands/RemoveCommand.java

```diff
@@ -19,85 +19,85 @@ public RemoveCommand() {
     /**
      * Executes the command.
      *
-     * @param ui The ui object to interact with user.
-     * @param parser The parser object to parse user inputs.
-     * @param ingredientList The ingredientList object.
-     * @param shoppingList The shoppingList object.
+     * @param ui             The ui object to interact with user.
```

Comment:
can remove the extra spaces here

Reply:
The space is actually part of the style because you get it when you hit:
`ctrl + alt + l`

We should be doing this shortcut for every java file for maximum correctness.